a.core_concepts.md

```diff
@@ -17,7 +17,7 @@ kubernetes.io > Documentation > Tasks > Access Applications in a Cluster > [Use
 <p>
 
 ```bash
-kubectl create namespace mynamespace
+kubectl create ns mynamespace
```

Comment:
There is enough CLI typing in this exam :)

Reply:
Agreed, maybe add 'namespace' as a comment? It will be good for people that are new to Kubernetes to be familiar with both options (small one to write, big one to recognize)